README.md

```diff
@@ -374,6 +374,185 @@ A series of examples are included as part of the tests, see
+            </artifactItems>
+            <overWriteReleases>false</overWriteReleases>
+            <overWriteSnapshots>true</overWriteSnapshots>
+          </configuration>
+        </execution>
+      </executions>
+    </plugin>
+  </plugins>
+</build>
+```
+
+Note that since we're copying the exporter in the package phase, you will need to run your QA tests
+in the `integration-test` phase. To simplify this, we recommend you use the Maven
+[failsafe](https://maven.apache.org/surefire/maven-failsafe-plugin/) plugin.
+
+Once done, you have everything you need to write an integration test in the `exporter-qa` module.
+For example:
+
+```java
+package com.acme.exporter.qa;
+
+import io.zeebe.containers.ZeebeContainer;
+import org.junit.jupiter.api.Test;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+import org.testcontainers.utility.MountableFile;
+
+@Testcontainers
+public final class MyExporterIT {
+  @Container
+  private final ZeebeContainer zeebe = new ZeebeContainer()
+    .withCopyFileToContainer(MountableFile.forClasspathResource("exporter.jar", 0775), "/usr/local/zeebe/exporters/exporter.jar")
+    .withEnv("ZEEBE_BROKER_EXPORTERS_ACME_CLASSNAME", "com.acme.exporter.Exporter")
+    .withEnv("ZEEBE_BROKER_EXPORTERS_ACME_JARPATH", "/usr/local/zeebe/exporters/exporter.jar")
+    .withEnv("ZEEBE_BROKER_EXPORTERS_ACME_ARGS_0_PROP", "foo");
+
+  @Test
+  void shouldExportData() {
+    // write your test here
+  }
+}
+```
+
+#### Loading Docker image
+
+The idea here is to prepare a Docker image which already contains your packaged exporter JAR, so no
+file mounting or packaging is necessary in advance. However, you do need to re-package your Docker
+image after every code change to your exporter before running your QA test.
+
+Here are some projects which do exactly this:
+
+- [Hazelcast Exporter](https://github.com/camunda-community-hub/zeebe-hazelcast-exporter/blob/main/exporter/src/test/java/io/zeebe/hazelcast/testcontainers/ZeebeTestContainer.java)
+- [Redis Exporter](https://github.com/camunda-community-hub/zeebe-redis-exporter/blob/main/exporter/src/test/java/io/zeebe/redis/testcontainers/ZeebeTestContainer.java)
+
+Let's say your build pipeline has already built the image as `ghcr.io/acme/exporter`, and the
+exporter JAR is located at `/usr/local/zeebe/exporters/exporter.jar` in that image. Then you could
+write the following test:
+
+```java
+package com.acme.exporter.qa;
+
+import io.zeebe.containers.ZeebeContainer;
+import org.junit.jupiter.api.Test;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+import org.testcontainers.utility.DockerImageName;
+
+@Testcontainers
+public final class MyExporterIT {
+  private static final DockerImageName TEST_IMAGE = DockerImageName.parse("ghcr.io/acme/exporter");
+
+  @Container
+  private final ZeebeContainer zeebe = new ZeebeContainer(TEST_IMAGE)
```

Comment:
❓ But this only works if this image builts on top of the zeebe image right ? 🤔

Reply:
Yes, good point! It's what these projects do, i'll update the docs